sklearn/linear_model/tests/test_logistic.py

```diff
@@ -443,11 +444,12 @@ def test_ovr_multinomial_iris():
     n_samples, n_features = train.shape
 
     # Use pre-defined fold as folds generated for different y
-    cv = StratifiedKFold(target, 3)
-    clf = LogisticRegressionCV(cv=cv)
+    cv = StratifiedKFold(3)
+    predefined_folds = list(cv.split(train, target))
```

Comment:
I changed it so to make it aligned with the comment... (which says predefined folds are used here...) 

Do I simply change the `cv=Stratified..` line alone here?